### ISSUE TITLE
Inserting new images inside the existing gallery

### ISSUE THREAD

**Issue by Rynaro** [volmer/bootsy#203]
Hello again,

(There's no wiki or mail list about bootsy, and I have no idea where I post my doubts.)

In `new action`  of my controller, Bootsy allows me to upload local files to store inside my application. And  the `bootsy_image_gallery` columns seems ok. All fields is populated properly, like this (real) example:
`#<Bootsy::ImageGallery id: 43, bootsy_resource_id: 11, bootsy_resource_type: "Doc", created_at: "2016-05-02 18:56:21", updated_at: "2016-05-02 18:56:48">`

I imagine, the `bootsy_resource_id` is my `doc.id`. (Right?)

My controller has `:bootsy_image_gallery_id` strong parameter, and my model has `include Bootsy::Container`

But when I call `edit action` of Doc, my Bootsy cannot open gallery, only shows the URL `http://` to insert images.

Thanks in advance. :smile:

**Comment by volmer** [comment]
It's hard to help without having an app that reproduces the error. My best guess is that you probably have an issue with the form on the edit page.

**Comment by Rynaro** [comment]
The application is a legacy application that uses a lot of gems and (some) integration with cloud services, now I'm in process to migrate the application to use local storage, and doesn't depends of cloud services anymore. 
I'm reporting here the major "difficulties" here. Because I don't know if it is a common behavior, a issue, or is a casual situation that other people face in use of Bootsy.
I really appreciate this gem. :smiley:  And I know, the legacy application has a many ways to conflicts with Bootsy (and some others gems), and I only report here, some behavior that I really have doubts about it.
Now, I'll investigate the app looking for the conflicts. (I only needed to know, if the gallery is available in edit action, and looking in the sources I found it)
Thanks.
